### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by agrojean-ledger**

### CODE REVIEW

File: src/ui/ui_idle_menu_nbgl.c [hunk 1]
```diff
@@ -29,9 +29,9 @@ enum {
     SWITCH_ALLOW_HASH_TX_TOKEN
 };
 
-#define NB_INFO_FIELDS 2
-static const char* const infoTypes[] = {"Version", "Tron App"};
-static const char* const infoContents[] = {APPVERSION, "(c) 2022 Ledger"};
+#define NB_INFO_FIELDS 3
+static const char* const infoTypes[] = {"Version", "Developer", "Copyright"};
+static const char* const infoContents[] = {APPVERSION, "Klever", "(c) 2022 Ledger"};
```
Comment (tdejoigny-ledger) [inline src/ui/ui_idle_menu_nbgl.c:34]: 2023 ?

File: src/ui/ui_idle_menu_nbgl.c
Comment (tdejoigny-ledger) [inline src/ui/ui_idle_menu_nbgl.c:1]: In 
`  nbgl_useCaseHome("Tron",
                     &C_stax_app_tron_64px,
                     "This app confirms actions on\nthe Tron network.",
                     true,
                     displaySettingsMenu,
                     onQuitCallback);`

replace `"This app confirms actions on\nthe Tron network."` by `NULL`

File: src/ui/ui_review_menu_nbgl.c [hunk 2]
```diff
@@ -149,45 +149,46 @@ static void prepareTxInfos(ui_approval_state_t state, bool data_warning) {
...
         case APPROVAL_TRANSFER:
+            PRINTF("TITI\n");
             txInfos.fields[0].item = "Amount";
             txInfos.fields[0].value = (const char *) G_io_apdu_buffer;
             txInfos.fields[1].item = "Token";
             txInfos.fields[1].value = fullContract;
             txInfos.fields[2].item = TRC20ActionSendAllow;
             txInfos.fields[2].value = toAddress;
-            txInfos.fields[3].item = "From Address";
+            txInfos.fields[3].item = "From";
```
Comment (tdejoigny-ledger) [inline src/ui/ui_review_menu_nbgl.c:171]: Could be interesting to use `#define` for all common strings ("From", "Review transaction", ...)

File: src/ui/ui_idle_menu_nbgl.c (hunk 1 above)
Comment (agrojean-ledger) [reply src/ui/ui_idle_menu_nbgl.c:34]: Indeed.

File: src/ui/ui_idle_menu_nbgl.c
Comment (agrojean-ledger) [reply src/ui/ui_idle_menu_nbgl.c:1]: Indeed.

File: src/ui/ui_review_menu_nbgl.c (hunk 2 above)
Comment (agrojean-ledger) [reply src/ui/ui_review_menu_nbgl.c:171]: Indeed. It won't save memory as the macros are replaced during pre-processing but still good practice.

Comment (agrojean-ledger) [reply src/ui/ui_review_menu_nbgl.c]: Using `static const char *` for common strings instead of macros.

**Review by tdejoigny-ledger — APPROVED** [review]

File: src/ui/ui_idle_menu_nbgl.c [hunk 3]
```diff
@@ -48,8 +48,13 @@ void onQuitCallback(void) {
 
 static bool settingsNavCallback(uint8_t page, nbgl_pageContent_t* content) {
     if (page == 0) {
+        content->type = INFOS_LIST;
+        content->infosList.nbInfos = NB_INFO_FIELDS;
+        content->infosList.infoTypes = (const char**) infoTypes;
+        content->infosList.infoContents = (const char**) infoContents;
```
Comment (xchapron-ledger) [inline src/ui/ui_idle_menu_nbgl.c:54]:
```suggestion
        content->infosList.infoTypes = infoTypes;
        content->infosList.infoContents = infoContents;
```
down cast not necessary anymore.

**Review by xchapron-ledger — APPROVED** [review]

File: src/ui/ui_idle_menu_nbgl.c (hunk 3 above)
Comment (agrojean-ledger) [reply src/ui/ui_idle_menu_nbgl.c:54]: Done

**Pull request merged**